code/modules/antagonists/space_dragon/space_dragon.dm

```diff
@@ -62,12 +62,7 @@
 	return ..()
 
 /datum/antagonist/space_dragon/get_preview_icon()
-	var/icon/icon = icon('icons/mob/nonhuman-player/spacedragon.dmi', "spacedragon")
-
-	icon.Blend(COLOR_STRONG_VIOLET, ICON_MULTIPLY)
-	icon.Blend(icon('icons/mob/nonhuman-player/spacedragon.dmi', "overlay_base"), ICON_OVERLAY)
-
-	icon.Crop(10, 9, 54, 53)
+	var/icon/icon = icon('code/modules/unit_tests/screenshots/screenshot_antag_icons_spacedragon.png')
```

Comment:
won't this break, considering that the unit test screenshots folder is meant to be regenerated?
(also this seems to be a completely incorrect usage of the image)

Reply:
Ah, figured, so then where do you store antag icons?

Reply:
You don't store it; the code that you had before was the way it was done
specifically, you're meant to make icons of things you want in the icon, then change their position and `Blend` them into the primary icon. Look at how nukeops do it

Reply:
Well, I guess since this was custom made it wouldn't make sense to try to recreate it that way since I'd be adding in custom sprites anyway.

Maybe I could move the icon to a different spot?